Readme.md

```diff
@@ -84,7 +84,7 @@ See [`Codetour.md`](Codetour.md) for an explanation of the repository structure.
 ### Automatic recompilation
 
 If the steps above worked, running `make watch` will trigger a quick
-recompilation every time a source file has been modified.
+recompilation every time a source file has been modified. It will also trigger a lint watch which will catch all the styling errors in the code.
```

Comment:
Yeah it is still under development under the [issue 72](https://github.com/WebMemex/webmemex-extension/pull/87) . I was just waiting on review from @treora . If you get a second, please check it out and provide feedback.